torchvision/models/detection/roi_heads.py

```diff
@@ -261,14 +261,11 @@ def heatmaps_to_keypoints(maps, rois):
         height_correction = heights[i] / roi_map_height
         roi_map = F.interpolate(
             maps[i][:, None], size=(roi_map_height, roi_map_width), mode='bicubic', align_corners=False)[:, 0]
-        # roi_map_probs = scores_to_probs(roi_map.copy())
         w = roi_map.shape[2]
         pos = roi_map.reshape(num_keypoints, -1).argmax(dim=1)
 
         x_int = pos % w
-        y_int = (pos - x_int) // w
-        # assert (roi_map_probs[k, y_int, x_int] ==
-        #         roi_map_probs[k, :, :].max())
```

Comment:
These might have been intended as some kind of documentation - perhaps leave them here? They were introduced in https://github.com/pytorch/vision/pull/898

Reply:
I see a few other commented out code in the same PR, for example:  `# print('creating index...')`, I assume these would have been for testing locally. I am not sure if it's a good idea to have commented out code checked-in. Anyways, I'll leave it here, perhaps we should revisit it later.